src/App.css

```diff
@@ -24,9 +24,9 @@
 
 @keyframes App-logo-spin {
   from {
-    transform: rotate(0deg);
+    transform: rotate(360deg);
   }
   to {
-    transform: rotate(360deg);
+    transform: rotate(0deg);
```

Comment:
It's good to keep pull requests as small as possible and do only one thing.

I would have addressed styling changes in a separate pr and keep this pr only about JWTs and Routing - I may be being a little picky here though.

Reply:
Oh! Yeah, this is leftover from using Create-React-App, and we were using it to play with making the logo move... totally didn't mean for this to make it into the PR!! 😅